src/components/songs/Songs.js

```diff
@@ -0,0 +1,119 @@
+import React, {Component} from 'react';
+import {withStyles} from '@material-ui/core/styles';
+import {connect} from 'react-redux';
+import {FindSongFetch, SongsFetch} from '../../store/songs/actions'
+import Container from "@material-ui/core/Container";
+import Song from './Song'
+import SearchForm from "./SearchForm";
+
+const styles = theme => ({
+    container: {
+        marginTop: theme.spacing(2),
+        flexDirection: 'column',
+        alignItems: 'center',
+    }
+});
+//https://www.pluralsight.com/guides/how-to-implement-infinite-scrolling-with-reactjs
+
+class Songs extends Component {
+    state = {
+        songs: [],
+        loading: false,
+        page: 1,
+        prevY: 0,
+        limit: 2
+    };
+
+    getSongs(page) {
```

Comment:
I lefting a comment here just to indicate that later we need to refactor this method
It have several propblems:
1) why in get songs method you are trying to check if length is 0 
2) if response from request will be quick you will may have a problem of not correct state updating